js/closure-compiler-scripts/tslib.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * closure-compiler-friendly tslib
+ * copied from node_modules/tslib/tslib.js
+ * update as needed
```

Comment:
Include the license for this here?

Reply:
@wesm this file is [released](https://github.com/arv/text-encoding-utf-8/blob/master/LICENSE.md) under The Unlicense, which I think [you said before](https://github.com/graphistry/arrow/issues/4#issuecomment-322663059) was ok.

Reply:
Ah, maybe just a note saying "License: The Unlicense"